database/nakadi/10_data/04_tables/02_subscription.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS zn_data.subscription (
+  s_id varchar(255) NOT NULL PRIMARY KEY CHECK (s_id <> ''),
+  s_subscription_object jsonb NOT NULL,
+  CHECK (s_subscription_object->>'id' = s_id)
+);
+
+CREATE INDEX ON zn_data.subscription USING gin (s_subscription_object);
+
+CREATE UNIQUE INDEX ON zn_data.subscription ((s_subscription_object->>'owning_application'),
```

Comment:
I think you also will need some checks on the not null not empty for the fields used in that index.